lib/omniauth/strategies/realme.rb

```diff
@@ -18,29 +39,45 @@ def request_phase
         redirect req.create(saml_settings, 'SigAlg' => 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256')
       end
 
-      def callback_phase # rubocop:disable Metrics/AbcSize
+      def callback_phase # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/PerceivedComplexity
```

Comment:
Rubocop will be significantly happier with this method when the Rails session stuff is eventually removed